data/payload_data.py

```diff
@@ -0,0 +1,7 @@
+first_name = 'Harry'
+last_name = 'Potter'
+total_price = 777
```

Comment:
jeżeli ta wartość przyjmuje wartość dziesiętną, to warto dodać jakieś grosze ku chwale bardziej różnorodnych danych :)